tests/test_scene_rendering/simple_scenes.py

```diff
@@ -0,0 +1,8 @@
+from manim import *
+
+
+class SquareToCircle(Scene):
+    def construct(self):
+        square = Square()
+        circle = Circle()
+        self.play(Transform(square, circle))
```

Comment:
Question: why include only one of the example scenes?

Reply:
For the moment only one is needed, as I use it only for flags-testing.

Reply:
I think we should have all of them in the future, as they use different funcionalities.

Reply:
Of course. The system is made to completed in the future with new scenes, etc. It's totally flexible